src/pages/Beta/Pool/EarnWidget/Stake/index.tsx

```diff
@@ -145,6 +148,7 @@ const Stake = ({ version, onComplete, type, stakingInfo, combinedApr }: StakePro
           addTransaction(response, {
             summary: t('earn.depositLiquidity')
           })
+          queryClient.refetchQueries(['get-minichef-farms-v2', account])
```

Comment:
can you please try this once more ? This line didn't work for me when I did farm, It didn't updated info in left side.